src/EnergyPlus/GeneralRoutines.cc

```diff
@@ -366,13 +366,13 @@ void ControlCompOutput(EnergyPlusData &state,
                     ZoneInterHalf.MaxResult = 1.0;
                     ZoneInterHalf.MinResult = 0.0;
                     // If between the max and mid set to new flow and raise min to mid
-                } else if ((ZoneController.SetPoint < ZoneInterHalf.MaxResult) && (ZoneController.SetPoint >= ZoneInterHalf.MidResult)) {
+                } else if (ZoneController.SetPoint >= ZoneInterHalf.MidResult) {
```

Comment:
[src/EnergyPlus/GeneralRoutines.cc:369]:(style),[knownConditionTrueFalse],Condition 'ZoneController.SetPoint<ZoneInterHalf.MaxResult' is always true